src/nav/infrastructure.yml

```diff
@@ -193,6 +193,8 @@ pages:
             path: /docs/infrastructure/prometheus-integrations/integrations-list/ingress-controller-integration
           - title: Redis integration
             path: /docs/infrastructure/prometheus-integrations/integrations-list/redis-integration
+          - title: Redpanda integration 
```

Comment:
The same comment as the previous one. Are you sure that this new page goes here?